flake.nix

```diff
@@ -1,6 +1,6 @@
 {
   inputs = {
-    holonix.url = "github:holochain/holonix?ref=main-0.4";
```

Comment:
Oh, I only saw this now. Was this change intentional? Shouldn't we use `main-0.4` on the `main-0.4` branch?

Reply:
ah, it got reverted again, nvm